<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="HABEEBUNNISA" failures="0" tests="3" name="com.saucedemo.Tests.RemoveFromCartTest" time="1.574" errors="1" timestamp="2024-05-26T18:30:51 IST" skipped="2">
  <testcase classname="com.saucedemo.Tests.RemoveFromCartTest" name="RemoveFromCart" time="0.000">
    <skipped/>
  </testcase> <!-- RemoveFromCart -->
  <system-out/>
  <testcase classname="com.saucedemo.Tests.RemoveFromCartTest" name="Login" time="1.574">
    <error message="java.io.IOException: Output closed
Build info: version: &#039;4.18.1&#039;, revision: &#039;b1d3319b48&#039;
System info: os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;17.0.6&#039;
Driver info: driver.version: ChromiumDriver" type="org.openqa.selenium.WebDriverException">
      <![CDATA[org.openqa.selenium.WebDriverException: java.io.IOException: Output closed
Build info: version: '4.18.1', revision: 'b1d3319b48'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.6'
Driver info: driver.version: ChromiumDriver
at org.openqa.selenium.remote.http.jdk.JdkHttpClient$5.send(JdkHttpClient.java:305)
at org.openqa.selenium.remote.http.jdk.JdkHttpClient$5.close(JdkHttpClient.java:324)
at org.openqa.selenium.devtools.Connection.close(Connection.java:224)
at org.openqa.selenium.devtools.DevTools.close(DevTools.java:60)
at java.base/java.util.Optional.ifPresent(Optional.java:178)
at org.openqa.selenium.remote.RemoteWebDriver.quit(RemoteWebDriver.java:427)
at org.openqa.selenium.chromium.ChromiumDriver.quit(ChromiumDriver.java:432)
at com.saucedemo.Base.ReportListener.onTestSuccess(ReportListener.java:49)
at org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:100)
at org.testng.internal.TestInvoker.runTestResultListener(TestInvoker.java:219)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:656)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.io.IOException: Output closed
at java.net.http/jdk.internal.net.http.websocket.WebSocketImpl.sendClose(WebSocketImpl.java:313)
at org.openqa.selenium.remote.http.jdk.JdkHttpClient$5.lambda$send$2(JdkHttpClient.java:284)
at org.openqa.selenium.remote.http.jdk.JdkHttpClient$5.send(JdkHttpClient.java:295)
... 27 more
]]>
    </error>
  </testcase> <!-- Login -->
  <system-out/>
  <testcase classname="com.saucedemo.Tests.RemoveFromCartTest" name="AddToCart" time="0.000">
    <skipped/>
  </testcase> <!-- AddToCart -->
  <system-out/>
</testsuite> <!-- com.saucedemo.Tests.RemoveFromCartTest -->
